float32[] velocities # the velocities of each of the joints
---
uint32 commandid # unique command id to be used for later referencing
time stamp # expected time when robot will start velocity
